spec/linter-stylelint-spec.js

```diff
@@ -171,4 +196,5 @@ describe('The stylelint provider for Linter', () => {
       })
     );
   });
+  
```

Comment:
Heh, this was actually the cause of the build failures as this wasn't passing the lint checks :stuck_out_tongue:.